content/docs/2.0/migration.md

```diff
@@ -6,6 +6,8 @@ title = "Migration Guide"
 
 Please note that you **can not** run both KEDA v1 and v2 on the same Kubernetes cluster. You need to [uninstall](../../1.5/deploy) KEDA v1 first, in order to [install](../deploy) and use KEDA v2.
 
+> 💡 **NOTE:** If you are using Helm, you will need to delete the v1 CRDs manually
```

Comment:
I'd make this note more generic. Something like, when uninstalling KEDA v1 make sure v1 CRDs are uninstalled from the cluster as well.